bin/table_georges_comparison

```diff
@@ -102,14 +111,64 @@ def write_tables(counts, tex_filename, with_preamble=False):
             row = [STYLE_SYMBOLS[style]] + \
                       ['%.2f\\%%' % (float(counts[machine][style]) /
                                      float(counts[machine]['total ' + style]) * 100.0)
-                       for machine in machines]
+                       for machine in sorted(machines)]
             fp.write('%s\\\\ \n' % '&'.join(row))
         fp.write(end_table())
         if with_preamble:
             fp.write('\\end{table*}\n')
             fp.write(end_document())
 
 
+def _reformat(word):
+    """Reformat word as a LaTeX macro name.
+    Removes spaces and underscores. Translates numbers to words. Lower cases.
+    """
+    for number in _NUMBERS:
+        word = word.replace(str(number), _NUMBERS[number])
+    word = word.replace(' ', '')
```

Comment:
This line and the one below may be clearer as `word = word.translate(None, " _")`?